### DISCUSSION TITLE
Provide more key shortcut options for Interactive mode

### DISCUSSION THREAD

**Discussion by rampatra** [Softal-io/presentify-community#3] [Ideas]
Provide more key shortcut options for Interactive mode. Currently, only the Fn key is the one set.

**Comment by danyindrawan** [comment]
Yes! We need that..

**Comment by rampatra** [comment]
Any ideas for this? What other keys would be good for this?

**Comment by danyindrawan** [comment]
I think "control" is the possible option.
Or, let the user be able to customize themselves would be MUCH BETTER.

**Comment by rampatra** [comment]
Presentify does not provide a way to customize the key binding for Interactive Mode on purpose. Doing this will make Presentify ask for permissions (listen to all key events) that many users will be reluctant to give. Having it set to the Fn key makes Presentify work without asking for any such permissions.

Having said that, I am planning to provide a fixed set of options (pre-defined) in the Preferences where the user can choose a different key if, for some reason, the Fn key doesn't work.

**Comment by danyindrawan** [comment]
Ok, thanks for the update.

**Comment by rampatra** [comment]
The latest update allows the user to choose between Fn and Caps lock keys. The former requires you to press and hold while the latter requires you to press just once.
